lib/array.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+module.exports = function(api) {
```

Comment:
It has just occurred to me: why aren't these exported functions arrow functions?